src/main/java/sirius/biz/importer/format/FieldDefinition.java

```diff
@@ -239,6 +240,7 @@ public FieldDefinition withLabel(String constantLabel) {
      */
     public FieldDefinition withLabel(Supplier<String> labelFunction) {
         this.label = labelFunction;
+        addAlias(getLabel());
```

Comment:
This will call the label function right away instead of when the user context is all set-up and the proper language is known.

If we _always_ create a new FieldDefinition, we can get away with this - but then we can simply accept a translated string where. Otherwise calling getLabel here is too early which is why it was moved into expandAliases...